model_analyzer/config/input/config_command.py

```diff
@@ -95,40 +96,137 @@ def set_config_values(self, args):
+
+    def _check_search_mode(self, args: Namespace,
+                           yaml_config: Optional[Dict[str, List]]) -> None:
+        if self._get_config_value('run_config_search_mode', args,
+                                  yaml_config) != 'quick':
+            raise TritonModelAnalyzerException(
+                f'\nConcurrent profiling of models is only supported in quick search mode.'
+                '\nPlease use quick search mode or disable concurrent model profiling.'
+            )
+
+    def _check_for_quick_search_incompatability(
+            self, args: Namespace, yaml_config: Optional[Dict[str,
+                                                              List]]) -> None:
+        if self._get_config_value('run_config_search_mode', args,
+                                  yaml_config) != 'quick':
+            return
+
+        self._check_no_search_values(args, yaml_config)
+        self._check_no_list_values(args, yaml_config)
+
+    def _check_no_search_values(self, args: Namespace,
+                                yaml_config: Optional[Dict[str, List]]) -> None:
+        max_concurrency = self._get_config_value(
+            'run_config_search_max_concurrency', args, yaml_config)
+        min_concurrency = self._get_config_value(
+            'run_config_search_min_concurrency', args, yaml_config)
+        max_instance = self._get_config_value(
+            'run_config_search_max_instance_count', args, yaml_config)
+        min_instance = self._get_config_value(
+            'run_config_search_min_instance_count', args, yaml_config)
+        max_batch_size = self._get_config_value(
+            'run_config_search_max_model_batch_size', args, yaml_config)
+        min_batch_size = self._get_config_value(
+            'run_config_search_min_model_batch_size', args, yaml_config)
+
+        if max_concurrency or min_concurrency:
+            raise TritonModelAnalyzerException(
+                f'\nProfiling of models in quick search mode is not supported with min/max concurrency search values.'
+                '\nPlease use brute search mode or remove concurrency search values.'
+            )
+        if max_instance or min_instance:
+            raise TritonModelAnalyzerException(
+                f'\nProfiling of models in quick search mode is not supported with min/max instance search values.'
+                '\nPlease use brute search mode or remove instance search values.'
+            )
+        if max_batch_size or min_batch_size:
+            raise TritonModelAnalyzerException(
+                f'\nProfiling of models in quick search mode is not supported with min/max batch size search values.'
+                '\nPlease use brute search mode or remove batch size search values.'
+            )
+
+    def _check_no_list_values(self, args: Namespace,
+                              yaml_config: Optional[Dict[str, List]]) -> None:
+        concurrency = self._get_config_value('concurrency', args, yaml_config)
```

Comment:
would this catch per-model concurrencies or batch sizes in a yaml config?

Reply:
No, I don't think it would. I'll look into creating that test, though I'm not exactly sure how to look for that condition yet.

Reply:
Added per model checking